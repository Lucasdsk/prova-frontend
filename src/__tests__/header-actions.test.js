import React from 'react'
import HeaderActions from './../components/header-actions'

describe("<HeaderActions />", () => {
  it("Render HeaderActions", () => {
    const HeaderActionsComponent = shallow(<HeaderActions />)
    expect(HeaderActionsComponent).toMatchSnapshot()
  })

  it("Number of notifications should match with the value passed by prop 'notificationCount'", () => {
    const HeaderActionsComponent = shallow(<HeaderActions notificationCount={3} />)
    expect(HeaderActionsComponent.find('.notifications-icon__counter').text()).toBe('3')
  })
})