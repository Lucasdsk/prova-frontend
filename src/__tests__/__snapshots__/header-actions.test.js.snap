// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderActions /> Render HeaderActions 1`] = `
<div
  className="header-actions"
>
  <FontAwesome
    className="header-actions__icon user"
    name="user"
  />
  <FontAwesome
    className="header-actions__icon comment"
    name="comment-o"
  />
  <span
    className="header-actions__icon notifications-icon"
  >
    <FontAwesome
      className="header-actions__icon"
      name="bell-o"
    />
    <span
      className="notifications-icon__counter"
    />
  </span>
</div>
`;
